.github/workflows/contracts.yml

```diff
@@ -282,53 +282,53 @@ jobs:
       - name: Check formatting
         run: yarn format
 
-  # The check is commented out until slither releases new version, with fix for
-  # https://github.com/crytic/slither/issues/1106.
-  # contracts-slither:
-  #   needs: contracts-detect-changes
-  #   if: |
-  #     github.event_name == 'push'
-  #       || needs.contracts-detect-changes.outputs.path-filter == 'true'
-  #   runs-on: ubuntu-latest
-  #   defaults:
-  #     run:
-  #       working-directory: ./solidity
-  #   steps:
-  #     - uses: actions/checkout@v2
-
-  #     - uses: actions/setup-node@v2
-  #       with:
-  #         node-version: "14.x"
-  #         cache: "yarn"
-  #         cache-dependency-path: solidity/yarn.lock
-
-  #     - uses: actions/setup-python@v2
-  #       with:
-  #         python-version: 3.8.5
-
-  #     - name: Install Solidity
-  #       env:
-  #         SOLC_VERSION: 0.8.9 # according to solidity.version in hardhat.config.ts
-  #       run: |
-  #         pip3 install solc-select
-  #         solc-select install $SOLC_VERSION
-  #         solc-select use $SOLC_VERSION
-
-  #     - name: Install Slither
-  #       env:
-  #         SLITHER_VERSION: 0.8.2
-  #       run: pip3 install slither-analyzer==$SLITHER_VERSION
-
-  #     # We need this step because the `@keep-network/tbtc` which we update in
-  #     # next steps has a dependency to `@summa-tx/relay-sol@2.0.2` package, which
-  #     # downloads one of its sub-dependencies via unathenticated `git://`
-  #     # protocol. That protocol is no longer supported. Thanks to this step
-  #     # `https://` is used instead of `git://`.
-  #     - name: Configure git to don't use unauthenticated protocol
-  #       run: git config --global url."https://".insteadOf git://
-
-  #     - name: Install dependencies
-  #       run: yarn install
-
-  #     - name: Run Slither
-  #       run: slither --hardhat-artifacts-directory build .
+  contracts-slither:
+    needs: contracts-detect-changes
+    # TODO: remove 'pull_request' condition before merging PR to main
```

Comment:
We should be good to drop this now @michalinacienciala. 

Can we undraft this PR?

Reply:
Undrafted, we should be ok to merge